functions/graphql/index.js

```diff
@@ -70,9 +70,9 @@ function configureServer() {
       // USER
       // const token = await User.getToken('participantd@example.com', 'Test1234');
       // CENTER
-      const token = "123456789012345678901234567890123456"; // Must be a 36 long string that matches the apiToken object on a center
+      // const token = "123456789012345678901234567890123456"; // Must be a 36 long string that matches the apiToken object on a center
       // DIRECTOR
-      // const token = await User.getToken('directora@example.com', 'Test1234');
+      const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```

Comment:
Let's make sure these get commented out before committing. This burned me yesterday and I had no idea what was going on for a little while. 😂
```suggestion
      // const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```